plugins/systemd/systemd.plugin.zsh

```diff
@@ -20,3 +20,14 @@ alias sc-mask-now="sc-mask --now"
 alias scu-enable-now="scu-enable --now"
 alias scu-disable-now="scu-disable --now"
 alias scu-mask-now="scu-mask --now"
+
+function sc-prompt-status {
+  for UNIT in $@
```

Comment:
Use `local` to make variables only visible inside this function. That prevents polluting the global variable namespace.